.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
Rockets haven't been necessary (barring having some characters in the key) since Ruby 2.0 I believe.